main/commons/src/main/java/org/cryptomator/common/vaults/Vault.java

```diff
@@ -230,8 +230,7 @@ public StringBinding displayableNameProperty() {
 	}
 
 	public String getDisplayableName() {
```

Comment:
Maybe rename this to `getDisplayName()`, too?

Reply:
Renamed all methods to something with "displayName" in https://github.com/cryptomator/cryptomator/pull/1318/commits/a8cb015a06639513a58f80617efa9a9b706b070f